smartsim/_core/mli/infrastructure/storage/backbone_feature_store.py

```diff
@@ -316,14 +326,15 @@ def _broadcast(self, timeout: float = 0.001) -> int:
 
     def send(self, event: EventBase, timeout: float = 0.001) -> int:
         """Implementation of `send` method of the `EventPublisher` protocol. Publishes
-        the supplied event to all registered broadcast consumers
-
-        :param event: an event to publish
-        :param timeout: maximum time to wait (in seconds) for messages to send
-        :returns: the number of events successfully published
-        :raises ValueError: if event serialization fails
-        :raises KeyError: if channel fails to attach using registered descriptors
-        :raises SmartSimError: if any unexpected error occurs during send"""
+        the supplied event to all registered broadcast consumers.
+
+        :param event: An event to publish
+        :param timeout: Maximum time to wait (in seconds) for messages to send
+        :returns: The number of events successfully published
```

Comment:
Just wanted to note that we are inconsistent with the usage of `:returns:` vs `:return:`. Both are fine for Sphinx, anyhow.

Reply:
I changed them all to be `:returns:` because that seems to be used more!